### DISCUSSION TITLE
GSOC: A Reference Implementation for concore Library in Java or Julia

### DISCUSSION THREAD

**Discussion by AbhinavMir** [ControlCore-Project/concore#60] [Ideas]
Hey! I was considering applying for '23 GSOC to work on this project and I'm considering taking an interfacing route where I'll probably use JuliaPy to bind Pythonic functions in Concore to Julia. Would this be a solid approach? Is there a more direct approach (rewriting in Julia) in mind?

**Comment by Ahmed89shaaban** [comment]
I hope to be agood idea to find new communication skills and collect data to renew applications and better

**Comment by ayushrai206** [comment]
Hello everyone, I am Ayushi, an Open Source enthusiast with a strong hand in Java, I would like to work on this project for GSOC proposal and would request the mentors for guidance on how to get started.

**Comment by harshal-rembhotkar** [comment]
Hii @mayureshkothare and @markgarnold  sir,

I am Harshal Rembhotkar, a 4th-semester Engineering student. I am excited to join your organization. My tech stacks are Java, HTML, CSS, version control systems, and technologies such as Docker and Kubernetes.
 I am particularly interested in contributing to Concore project:"a Reference Implementation for concore Library in Java or Julia" in Alaska organization under your mentorship .I am completely enthusiastic about taking part in GSOC 2024 in Alaska and giving my best in it!
Thank you..

Linkedin: https://www.linkedin.com/in/harshal-rembhotkar/
Github: https://github.com/Harshal-Rembhotkar

**Comment by harshal-rembhotkar** [comment]
Hi @markgarnold and @mayureshkothare , How can we submit a draft proposal? Do we have to email it?
